lib/issuer.js

```diff
@@ -123,15 +116,18 @@ api.issue = function(actor, credential, options, callback) {
         actor, PERMISSIONS.CREDENTIAL_ISSUE,
         {resource: credential, translate: 'issuer'}, callback);
     },
-    sign: ['checkPermission', function(callback) {
-      if(!options.privateKey) {
+    getSigningKey: ['checkPermission', function(callback) {
+      getCredentialSigningKey(credential, callback);
+    }],
+    sign: ['getSigningKey', function(callback, results) {
+      if(!results.getSigningKey) {
         return callback(null, credential);
       }
       // digitally-sign credential
       jsigs.sign(credential, {
         algorithm: 'LinkedDataSignature2015',
-        privateKeyPem: options.privateKey,
-        creator: options.publicKeyId
+        privateKeyPem: results.getSigningKey.privateKey.privateKeyPem,
```

Comment:
It's there! the last commit isn't getting shown in the overview for whatever reason, but if you go to Files Changed you'll see it.